app/controllers/hyrax/file_sets_controller.rb

```diff
@@ -254,12 +225,6 @@ def empty_file?(file)
         (file.respond_to?(:tempfile) && file.tempfile.size == 0) || (file.respond_to?(:size) && file.size == 0)
       end
 
-      # this is provided so that implementing application can override this behavior and map params to different attributes
```

Comment:
This description, the method name and the method content seem unrelated to each other.  It being a private method means it is **not** part of the public contract of the class, regardless of the comment's claim.  The approach that says (several times over in this class) "override this private method" is a fundamental OO error we should avoid.  Basic OO approaches or well-known Rails callbacks would be appropriate directions for Hyrax to point users towards instead of overriding private methods.